website/pages/payments.html

```diff
@@ -67,27 +65,21 @@
         <th>Amount</th>
     </tr>
     </thead>
-      {{ for(var p in it.stats.pools[pool].payments) { }}
-<tr>
-          <td>
-          {{if (String(it.stats.pools[pool].name).startsWith("zcash")) { }}
-          <a href="https://explorer.zcha.in/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("zclassic")) { }}
-            <a href="https://classic.zcha.in/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("hush")) { }}
-            <a href="https://explorer.myhush.org/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else if (String(it.stats.pools[pool].name).startsWith("zen")) { }}
-            <a href="http://node1.zenchain.info:8886/tx/{{=it.stats.pools[pool].payments[p].txid}}"  title="View transaction" target="_blank">{{=it.stats.pools[pool].payments[p].blocks}}</a>
-          {{ } else { }}
-          {{=it.stats.pools[pool].payments[p].blocks}}
-          {{ } }}
-          </td>
-          <td>{{=readableDate(it.stats.pools[pool].payments[p].time)}}</td>
-          <td>{{=it.stats.pools[pool].payments[p].miners}}</td>
-          <td>{{=Math.round(it.stats.pools[pool].payments[p].shares)}}</td>
-          <td>{{=it.stats.pools[pool].payments[p].paid}} {{=it.stats.pools[pool].symbol}}</td>
-      </tr>
-      {{ } }}
-  </table>
+		{{ for(var p in it.stats.pools[pool].payments) { }}
+		<tr>
+			<td>
+			{{ if (it.poolsConfigs[pool].explorer && it.poolsConfigs[pool].explorer.txURL) { }}
```

Comment:
This looks good to me. A couple of requests:
- Swap out tabs for spaces (four spaces per tab)
- Shift the alignment for the following three lines one "tab" to the left

Other than that, I think this is ready for the config update. Please note, the config comment would need to state that the full url (excluding the block or tx id -- but including a trailing slash is needed)